README.md

```diff
@@ -154,13 +161,34 @@ Please note, that you cannot override default fields by the python logger (i.e.
 For example:
 
 
-```
+```python
 logger.info('Warning', extra={'extra_key':'extra_value'})
 ```
+#### Dynamic Extra Fields
+The following additional code example offers the same functionlites that availavle with the extra parameter to add additional fields to logs. The difference is that it uses logging filters so it will add the fields that are declared key-values from the extra dictionary to every log that's generated after adding the fliter. You can keep updating the logs with additional filters. 
```

Comment:
```suggestion
If you prefer, you can add extra fields to your logs dynamically, and not pre-defining them in the configuration.
This way, you can allow different logs to have different extra fields.
See the following code example:
```
I think something like this, to keep it simple. If you want, you can consult with Nico about how to phrase it.